### PR TITLE
Added a function to write job submission .sh script

### PR DESCRIPTION
**Pull request by johnboyington**
See commits below.

### CODE REVIEW

File: responseWriter.py [hunk 1]
```diff
@@ -29,19 +30,46 @@
...
+This function creates the source term for the bonner sphere mcnp input file
+
+It uses the cosine bin data from the given energy group to write the source distribution
+
+Inputs:
+    numb - the energy group number
+    ee - the energy group's lower bound (MeV)
+    eee - the energy group's upper bound (MeV)
+'''
 def Source(numb, ee, eee):
```
Comment (johnboyington) [inline responseWriter.py:71]: more appropriate variable names

File: responseWriter.py [hunk 2]
```diff
@@ -80,10 +113,14 @@ def Card2():
     H = ''
     H = open('card2.txt', 'r').read()
     return H
+
+#this function names the mcnp input file
 def Name(big, erg):
```
Comment (Wenkai-Fu) [inline responseWriter.py]: comments by Kevin

File: responseWriter.py [hunk 3]
```diff
@@ -29,19 +30,46 @@
...
+It uses the cosine bin data from the given energy group to write the source distribution
+
+Inputs:
+    numb - the energy group number
+    ee - the energy group's lower bound (MeV)
+    eee - the energy group's upper bound (MeV)
+'''
 def Source(numb, ee, eee):
+    #first value in SP card must be zero, so add a zero to the cosine data
     d = np.concatenate((np.array([0]), nfl[numb]), axis=0)
```
Comment (robertsj) [inline responseWriter.py:73]: nfl is too short---you're not using Fortran77!

File: responseWriter.py [hunk 4]
```diff
@@ -63,11 +91,16 @@ def Source(numb, ee, eee):
         H += '{}  {:10.5e} {:10.5e} {:10.5e} {:10.5e}\n'.format(card, d[ii * 4], d[ii * 4 + 1], d[ii * 4 + 2], d[ii * 4 + 3])
     return H
 
+#writes the sphere card for the mcnp input
 def Sphere(big):
     H = ''
```
Comment (robertsj) [inline responseWriter.py:98]: 96 does nothing

Comment (johnboyington) [inline responseWriter.py:98]: condense down to one line

File: responseWriter.py [hunk 5]
```diff
@@ -63,11 +91,16 @@ def Source(numb, ee, eee):
...
     H = '01  S   0 0 -0.9  {}    $Bonner Sphere\n'.format(big)
     return H
 
+'''
+The next 3 functions just read in the mcnp input file data for different cards
+that exist in text files in this directory. They are constant for each problem.
+'''
 def Card0():
```
Comment (johnboyington) [inline responseWriter.py:106]: change these functions similar to line 96

Comment (robertsj) [inline responseWriter.py:106]: Function names ought to be more descriptive

Comment (johnboyington) [reply responseWriter.py:106]: get rid of these functions

File: responseWriter.py [hunk 6]
```diff
@@ -95,11 +132,33 @@ def subFile(name):
...
     
 #set parameters
-erg = np.array(range(len(eb) - 9)) + 1
+erg = range(len(eb) - 1)
 dia = range(len(size))
 
+#loop through each energy group and sphere size
 for ii in dia:
     for jj in erg:
         s = ''
```
Comment (johnboyington) [inline responseWriter.py:166]: comment what's s & t?

File: responseWriter.py [hunk 7]
```diff
@@ -95,11 +132,33 @@ def subFile(name):
...
+
+#this function creates a bash file to submit all of the qsub files for each mcnp input
+def bashFile(diameter, energy):
+    H = '#!/bin/bash\n'
+    H += 'declare -a siz=('
+    for ii in diameter:
+        H += '{} '.format(ii)
+    H += ')\n'
+    H += 'declare -a erg=('
+    for jj in energy:
```
Comment (robertsj) [inline responseWriter.py:146]: I suggest `for e in energy`

File: responseWriter.py [hunk 8]
```diff
@@ -110,9 +169,11 @@ def subFile(name):
         s += Card2()
         t = ''
         t += subFile(Name(ii, jj))
-        with open('/home/john/bonner_sphere/mcnp/writer/output/{}.i'.format(Name(ii, jj)), 'w') as H:
+        with open('output/{}.i'.format(Name(ii, jj)), 'w') as H:
```
Comment (johnboyington) [inline responseWriter.py:174]: can use the same variable for each H.write

**Pull request closed without merging**